client/rest-high-level/src/main/java/org/elasticsearch/client/MLRequestConverters.java

```diff
@@ -612,6 +612,9 @@ static Request getDataFrameAnalytics(GetDataFrameAnalyticsRequest getRequest) {
                 params.putParam(PageParams.SIZE.getPreferredName(), pageParams.getSize().toString());
             }
         }
+        if (getRequest.getAllowNoMatch() != null) {
+            params.putParam(GetDataFrameAnalyticsRequest.ALLOW_NO_MATCH.getPreferredName(), Boolean.toString(getRequest.getAllowNoMatch()));
```

Comment:
Alternatively, you could use:
getRequest.getAllowNoMatch().toString()
as you know it is not null from line 615.

Reply:
We use `Boolean.toString` in the rest of the file so I'd rather keep it consistent.